agent/api/ecsclient/client_test.go

```diff
@@ -231,13 +235,34 @@ func TestSubmitContainerStateChangeLongReason(t *testing.T) {
 	}
 }
 
-func TestRegisterContainerInstance(t *testing.T) {
+func BuildAttributeList(caps []string, attributes map[string]string) []*ecs.Attribute {
```

Comment:
nit: Is there any reason for this to be publicly scoped?  I'd prefer it were private unless there's a specific reason.

Also, can you change `caps` to `capabilities`?